src/CachePlugin.php

```diff
@@ -225,6 +224,10 @@ private function getCacheControlDirective(ResponseInterface $response, $name)
      */
     private function createCacheKey(RequestInterface $request)
     {
+        if ('POST' === $request->getMethod()) {
+            return hash($this->config['hash_algo'], $request->getMethod().' '.$request->getUri().' '.$request->getBody());
```

Comment:
i would not consider that a BC break. chances are you replace the cache on a new deployment anyways, and even if not, its just a cache.